pkg/build/api/v1/types.go

```diff
@@ -249,7 +249,7 @@ type DockerBuildStrategy struct {
 type SourceBuildStrategy struct {
 	// From is reference to an ImageStream, ImageStreamTag, or ImageStreamImage from which
 	// the docker image should be pulled
-	From *kapi.ObjectReference `json:"from,omitempty" description:"reference to an image stream, image stream tag, or image stream image from which the Docker image should be pulled"`
+	From kapi.ObjectReference `json:"from" description:"reference to an image stream, image stream tag, or image stream image from which the Docker image should be pulled"`
```

Comment:
yeah i was going to make the same claim (that validation would/should have rejected it anyway)